admin/resources/js/stories/PoolCandidates.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { createClient } from "urql";
+import { action } from "@storybook/addon-actions";
+import PoolCandidatesTable from "../components/poolCandidate/PoolCandidatesTable";
+import fakePoolCandidates from "../fakeData/fakePoolCandidates";
+import {
+  CreatePoolCandidateInput,
+  User,
+  Classification,
+  OperationalRequirement,
+  Pool,
+  CmoAsset,
+  UpdatePoolCandidateInput,
+  PoolCandidate,
+  LanguageAbility,
+  WorkRegion,
+  PoolCandidateStatus,
+  SalaryRange,
+  UpdatePoolCandidateMutation,
+} from "../api/generated";
+import { CreatePoolCandidateForm } from "../components/poolCandidate/CreatePoolCandidate";
+import fakeUsers from "../fakeData/fakeUsers";
+import fakeClassifications from "../fakeData/fakeClassifications";
+import fakePools from "../fakeData/fakePools";
+import fakeCmoAssets from "../fakeData/fakeCmoAssets";
+import fakeOperationalRequirements from "../fakeData/fakeOperationalRequirements";
+import { UpdatePoolCandidateForm } from "../components/poolCandidate/UpdatePoolCandidate";
+import { currentDate } from "../components/form/formUtils";
+
+const poolCandidateData = fakePoolCandidates();
+// Its possible data may come back from api with missing data.
+
+const stories = storiesOf("Pool Candidates", module);
+
+stories.add("Pool Candidates Table", () => (
+  <PoolCandidatesTable poolCandidates={poolCandidateData} />
+));
+
+// TODO: Pool Candidates Table API
+// const client = createClient({
+//   url: "http://localhost:8000/graphql",
+// });
+// stories.add("Pool Candidates Table with API data", () => (
+//   <ClientProvider client={client}>
+//     <PoolCandidatesTable />
+//   </ClientProvider>
+// ));
+
+stories.add("Create Pool Candidate Form", () => (
```

Comment:
I'd still like to see stories for the CreatePoolCandidate and UpdatePoolCandidate components which are connected directly to the API. They help ensure everything's actually working.